src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperParseTreeRewriter.cs

```diff
@@ -67,6 +73,10 @@ private void RewriteTags(Block input, RewritingContext context)
                         continue;
                     }
                 }
+                else
+                {
+                    ValidateParentTagHelperAllowsContent(child, context.ErrorSink);
```

Comment:
Suspect this will result in errors about whitespace e.g. the following won't be valid but should be
``` html
  <parent>
    <allowedChild />
  </parent>
```
Is that correct?